src/core/providers/strategies/LedgerProviderStrategy/helpers/initializeLedgerProvider.ts

```diff
@@ -1,43 +1,42 @@
+import { LedgerConnectStateManager } from 'core/managers/internal/LedgerConnectStateManager/LedgerConnectStateManager';
 import { getIsLoggedIn } from 'core/methods/account/getIsLoggedIn';
-import { LedgerConnectEventsEnum } from '../types';
 import { getLedgerErrorCodes } from './getLedgerErrorCodes';
 import { getLedgerProvider } from './getLedgerProvider';
-import {
-  LedgerEventBusType,
-  LedgerConnectStateManagerType
-} from '../types/ledgerProvider.types';
 
 type InitializeLedgerProviderType = {
-  eventBus?: LedgerEventBusType;
-  manager?: LedgerConnectStateManagerType | null;
+  manager: LedgerConnectStateManager | null;
   resolve: (value: Awaited<ReturnType<typeof getLedgerProvider>>) => void;
   reject: (reason?: string) => void;
 };
 
 const failInitializeErrorText = 'Check if the MultiversX App is open on Ledger';
 
 export const initializeLedgerProvider = async ({
-  eventBus,
   manager,
   resolve,
   reject
 }: InitializeLedgerProviderType) => {
   const shouldInitiateLogin = !getIsLoggedIn();
 
   // Calls itself to handle retry logic if the user needs to reconnect to the Ledger provider.
-  const onRetry = () =>
-    initializeLedgerProvider({ eventBus, manager, resolve, reject });
+  const onRetry = () => initializeLedgerProvider({ manager, resolve, reject });
```

Comment:
This could be called `handleRetry` maybe? Same with `onCancel` below. We usually use the `on` prefix when the method is passed as a property or argument from the caller.